src/main/java/seedu/address/logic/parser/AddStudentCommandParser.java

```diff
@@ -48,11 +50,12 @@ public class AddStudentCommandParser implements Parser<AddStudentCommand> {
     public AddStudentCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_GENDER, PREFIX_PHONE, PREFIX_EMAIL,
-                    PREFIX_ADDRESS, PREFIX_TAG, PREFIX_SUBJECT, PREFIX_CLASSES, PREFIX_ATTENDANCE);
+                    PREFIX_ADDRESS, PREFIX_TAG, PREFIX_SUBJECT, PREFIX_CLASSES, PREFIX_ATTENDANCE, PREFIX_NEXT_OF_KIN,
+                        PREFIX_EMERGENCY_CONTACT);
 
         // Ensure all required prefixes are present
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_GENDER, PREFIX_PHONE, PREFIX_EMAIL,
-                PREFIX_SUBJECT, PREFIX_CLASSES) || !argMultimap.getPreamble().isEmpty()) {
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_ATTENDANCE,
```

Comment:
PREFIX_GENDER is missing and PREFIX_ADDRESS should be removed.

I think you didn't merge with the latest version in the master branch.